### DISCUSSION TITLE
Lightcn里为什么是获取权重而不是嵌入

### DISCUSSION THREAD

**Discussion by primising** [RUCAIBox/RecBole#1891] [Q&A]
`def get_ego_embeddings(self):
        r"""Get the embedding of users and items and combine to an embedding matrix.

        Returns:
            Tensor of the embedding matrix. Shape of [n_items+n_users, embedding_dim]
        """
        user_embeddings = self.user_embedding.weight
        item_embeddings = self.item_embedding.weight
        ego_embeddings = torch.cat([user_embeddings, item_embeddings], dim=0)
        return ego_embeddings
